src/Symfony/Component/Routing/Loader/AttributeClassLoader.php

```diff
@@ -230,6 +239,19 @@ protected function addRoute(RouteCollection $collection, object $attr, array $gl
             } else {
                 $collection->add($name, $route, $priority);
             }
+            foreach ($attr->getAliases() as $aliasAttribute) {
+                if (\is_string($aliasAttribute)) {
```

Comment:
Inverting the condition (ie: `$aliasAttribute instanceof DeprecatedAliad`) might be easier to understand IMO.